db/migrate/20230220173322_create_users.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# Migration to create users groups join table
+
+class CreateUsers < ActiveRecord::Migration[7.0]
```

Comment:
- [ ] There's a duplicate users table. This causes issues when setting up your app locally. You'll kindly require to resolve this issue.

Reply:
- [ ] As mentioned by the previous Reviewer, this is a duplicate table. As one "users" table already created using devise. Please fix that.